### ISSUE TITLE
error loading plugin parentwork

### ISSUE THREAD

**Issue by 1DontEx1st** [beetbox/beets#3472]
Trying to use parentwork plugin for tagging classical music, any usage of _beet_ throws the following:

** error loading plugin parentwork:
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/beets/plugins.py", line 273, in load_plugins
    namespace = __import__(modname, None, None)
ModuleNotFoundError: No module named 'beetsplug.parentwork'

### Setup

* OS: latest linuxserver.io/beets docker container
* Python version: 3.8.1
* beets version: 1.4.9
* Turning off plugins made problem go away (yes/no): 
Kinda, but that also defeats the point.
My configuration (output of `beet config`) is:

```yaml
** error loading plugin parentwork:
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/beets/plugins.py", line 273, in load_plugins
    namespace = __import__(modname, None, None)
ModuleNotFoundError: No module named 'beetsplug.parentwork'


plugins: fetchart embedart convert scrub replaygain lastgenre chroma web inline parentwork
directory: /music_new
library: /config/musiclibrary.blb
art_filename: albumart
threaded: yes
original_date: no
per_disc_numbering: yes
convert:
    auto: no
    ffmpeg: /usr/bin/ffmpeg
    opts: -ab 320k -ac 2 -ar 48000
    max_bitrate: 320
    threads: 1
    dest:
    pretend: no
    format: mp3
    id3v23: inherit
    formats:
        aac:
            command: ffmpeg -i $source -y -vn -acodec aac -aq 1 $dest
            extension: m4a
        alac:
            command: ffmpeg -i $source -y -vn -acodec alac $dest
            extension: m4a
        flac: ffmpeg -i $source -y -vn -acodec flac $dest
        mp3: ffmpeg -i $source -y -vn -aq 2 $dest
        opus: ffmpeg -i $source -y -vn -acodec libopus -ab 96k $dest
        ogg: ffmpeg -i $source -y -vn -acodec libvorbis -aq 3 $dest
        wma: ffmpeg -i $source -y -vn -acodec wmav2 -vn $dest
    tmpdir:
    quiet: no
    embed: yes

    paths: {}
    no_convert: ''
    never_convert_lossy_files: no
    copy_album_art: no
    album_art_maxwidth: 0

paths:
    default: $albumartist/$album%aunique{}/$albumartist - $album%aunique{} - %if{$multidisc,CD$disc - }$track - $title
    singleton: $albumartist/$artist - $title
    comp: $#Compilations/$album%aunique{}/$album%aunique{} - $track - $artist - $title
    albumtype_soundtrack: $#Compilations/$album%aunique{}/$album%aunique{} - $track - $artist - $title
item_fields:
    multidisc: 1 if disctotal > 1 else 0

import:
    write: yes
    copy: yes
    move: no
    resume: ask
    incremental: yes
    quiet_fallback: skip
    timid: no
    log: /config/beet.log
lastgenre:
    auto: yes
    source: album
    whitelist: yes
    min_weight: 10
    count: 1
    fallback:
    canonical: no
    force: yes
    separator: ', '
    prefer_specific: no
embedart:
    auto: yes
    maxwidth: 0
    compare_threshold: 0
    ifempty: no
    remove_art_file: no
fetchart:
    auto: yes
    minwidth: 0
    maxwidth: 0
    enforce_ratio: no
    cautious: no
    cover_names:
    - cover
    - front
    - art
    - album
    - folder
    sources:
    - filesystem
    - coverart
    - itunes
    - amazon
    - albumart
    google_key: REDACTED
    google_engine: 001442825323518660753:hrh5ch1gjzm
    fanarttv_key: REDACTED
    store_source: no
replaygain:
    auto: no
    overwrite: no
    backend: command
    targetlevel: 89
    r128: [Opus]
    command: ''
    noclip: yes
scrub:
    auto: yes

replace:
    ^\.: _
    '[\x00-\x1f]': _
    '[<>:"\?\*\|]': _
    '[\xE8-\xEB]': e
    '[\xEC-\xEF]': i
    '[\xE2-\xE6]': a
    '[\xF2-\xF6]': o
    '[\xF8]': o
    \.$: _
    \s+$: ''
web:
    host: 0.0.0.0
    port: 8337
    cors: ''
    cors_supports_credentials: no
    reverse_proxy: no
    include_paths: no
pathfields: {}
album_fields: {}
chroma:
    auto: yes
```

**Comment by sampsyo** [comment]
The parentwork plugin was not available in 1.4.9; it was only added since then. We should really put out a new release soon, but in the mean time, feel free to run from source (see the FAQ).

**Comment by 1DontEx1st** [comment]
Thanks for the quick response! I changed from "latest" to "nightly" and no longer get the error. Now, to figure out how to actually use parentwork within "paths" :)
